Lib/ufo2ft/postProcessor.py

```diff
@@ -357,6 +360,27 @@ def _subroutinize_with_cffsubr(cls, otf, cffVersion):
 
         return cffsubr.subroutinize(otf, cff_version=cffVersion, keep_glyph_names=False)
 
+    def apply_fontinfo(self):
+        """Apply the fontinfo data from the DesignSpace variable-font's lib to
+        the compiled font."""
+        import copy
+
+        from ufo2ft.infoCompiler import InfoCompiler
+
+        logger.info("Applying variable-font info from DesignSpace lib")
+
+        # Create a temporary UFO and sets its fontinfo to the union of the main
+        # UFO's fontinfo and the DesignSpace variable-font’s info.
+        temp_ufo = type(self.ufo)()
+        temp_ufo.info = copy.copy(self.ufo.info)
+        for k, v in self.info.items():
+            setattr(temp_ufo.info, k, v)
```

Comment:
you could add a parameter to InfoCompiler's constructor that takes a dict (containing the fontinfo overrides) and then do the creation of this temp_ufo with the merged fontinfos inside the InfoCompiler's constructor; it'd keep the logic more self-contained and distinct from PostProcessor itself, and might make easier to write unit tests as well